r124/outref/ydb312_gtm8182c.txt

```diff
@@ -0,0 +1,46 @@
+Test A: INST3-INST4 jnlpool is down on INST3
+-------------------------------------------------
+
+# Create the DB with maximum key size set to 5 characters
+# Start INST1 INST2 replication
+==Executing MULTISITE_REPLIC 'START INST1 INST2'==
+Starting Primary Source Server in ##TEST_PATH##
+Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance2
+
+# Start INST3 INST4 replication
+==Executing MULTISITE_REPLIC 'START INST3 INST4'==
+Starting Primary Source Server in ##FILTERED##_REMOTE_TEST_PATH_/instance3
+Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance4
+
+# Run ydb312gtm8182c.m to trigger a KEY2BIG error in INST3 (should cause INST3 to freeze)
```

Comment:
Need a check of INST3 and INST1 freeze status to verify that only INST3 is frozen and INST1 is not frozen.